drivers/clock_control/Kconfig.nrf

```diff
@@ -128,6 +128,27 @@ config CLOCK_CONTROL_NRF_USES_TEMP_SENSOR
 endif # CLOCK_CONTROL_NRF_DRIVER_CALIBRATION
 endif # CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
 
+config CLOCK_CONTROL_NRF_HF_CALIBRATION
```

Comment:
what about `_HF_` -> `_HFINT_` ? Errata specifically mentions `HFINT` so maybe it is better to be more specific here, as HFXO applications seem to be unaffected.

Applies to all naming occurrences